python/rpmte-py.c

```diff
@@ -233,8 +257,12 @@ static struct PyMethodDef rpmte_methods[] = {
     {"Failed",	(PyCFunction)rpmte_Failed,	METH_NOARGS,
      "te.Failed() -- Return if there are any related errors."},
     {"Key",	(PyCFunction)rpmte_Key,		METH_NOARGS,
-     "te.Key() -- Return the associated opaque key aka user data\n\
+     "te.Key() -- Return the associated opaque retrieval key\n\
 	as passed e.g. as data arg ts.addInstall()"},
+    {"Priv",	(PyCFunction)rpmte_Priv,	METH_NOARGS,
+     "te.Priv() -- Return associated user data (if any)\n"},
+    {"SetPriv",	(PyCFunction)rpmte_SetPriv,	METH_O,
+     "te.Priv() -- Set associated user data (if any)\n"},
```

Comment:
`"te.SetPriv(userdata) -- Set associated user data (if any)\n"`

Note:
I prefer `userdata` instead of `priv`.
Example: `SetUserData ()`, `GetUserData ()`/`UserData ()`. Below the `priv`, I can imagine a something other.
And in the callback `void *userdata` instead of` void *priv`.
But that's my feeling.

Reply:
Fair enough, not sure what was supposed to be the idea behind using "priv" there. The only easy thing about names is picking a bad one :sweat_smile:

Changed to rpmte(Set)Userdata etc now, thanks for the feedback.